staging/src/k8s.io/apiserver/pkg/server/openapi/openapi_aggregator.go

```diff
@@ -0,0 +1,482 @@
+/*
```

Comment:
I plan to move OpenAPI stuff out as a library, so where this code exists doesn't matter to me, I will move it to api aggregator for now.